### DISCUSSION TITLE
Switching between postgresql in production and sqlite in development

### DISCUSSION THREAD

**Discussion by leo-petrucci** [prisma/prisma#3642] [Q&A]
I'm fairly new to Prisma but I'm really enjoying it so far. I've created my schema and it's working great. I'm ready to deploy it, but I can't figure out how to tell Prisma to use sqlite while developing/testing and using a remote Postgresql database while in production.

My initial idea was to simply switch ENV variables for both `provider` and `url`, but VS code tells me I can't use Environment Variables for `provider`.

The other way I can think of is to set the `schema` file in `package.json` and have two separate schemas for development/production. But that seems like a very roundabout way to do this.

What am I missing? What do you suggest?

**Comment by ryands17** [comment]
@creativiii 👋 
You can absolutely do that via multiple providers as explained [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-sources#multiple-providers).

**Reply by arthurfiorette** [reply]
Hey @ryands17, seems that this link is broken. Does prisma 2 still supports this?

**Reply by SteveChurch** [reply]
No, it seems not. `A Prisma schema can only have one data source.`

**Reply by patrick91** [reply]
yeah, this feature has been removed. Is there any workaround?

**Comment by rekliner** [comment]
@creativiii Please update your answer if you see this.  Unfortunately prisma has discontinued supporting different providers programmatically.  They suggest they may revisit the issue later but no promises.  Their suggestion for now is to use the same database in development as in production.  Yes, many people are disappointed.  https://github.com/prisma/prisma/issues/3834

**Reply by jharrell** [reply]
We have a few open issues ( #1274 and #2443 ) that cover a part of this space or at the very least mention it. I'd advise you to either 👍 those or open a new issue for this datasource-per-environment feature request

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
